java-optional/src/test/java/none/cvg/optional/TestKata1OptionalCreationAndFetchingValues.java

```diff
@@ -156,14 +156,14 @@ public void getValueFromOptionalForNonNullValue() {
 
         /*
          * TODO:
-         *  Replace the "null" to get the Optional has a null value.
+         *  Replace the "11" to get the Optional has a null value.
```

Comment:
Good catch ! Though this should be 10 and not 11. the inner "assertNotEquals()" will only throw is both the unexpected and actual are actually equal. It (correctly) does not throw a NoSuchElementException, which the test failure that the kata taker has to fix.